_extensions/odoo/translator.py

```diff
@@ -58,6 +68,10 @@ def __init__(self, builder, document):
         self.context = []
         self.section_level = 0
 
+        if int(sphinx.__version__.split('.')[0]) >= 2:
+            # TODO: do that by default in V14.0
+            # This preserve compatibility between sphinx 1.6 and 2.3
+            self.config = builder.config
```

Comment:
Either set it unconditionally or set IFF it's unset, no need to version-check. Or fix the places which access self.config to use self.builder.config. Or make self.config into a property.